program-test/tests/sysvar.rs

```diff
@@ -53,3 +57,72 @@ async fn get_sysvar() {
         .await
         .unwrap();
 }
+
+fn epoch_reward_sysvar_getter_process_instruction(
+    _program_id: &Pubkey,
+    _accounts: &[AccountInfo],
+    input: &[u8],
+) -> ProgramResult {
+    msg!("epoch_reward_sysvar_getter");
+
+    // input[0] == 0 indicates the bank is not in reward period.
+    // input[0] == 1 indicates the bank is in reward period.
+    if input[0] == 0 {
+        // epoch rewards sysvar should not exist for banks that are not in reward period
+        let epoch_rewards = EpochRewards::get();
+        assert!(epoch_rewards.is_err());
+    } else {
+        let _epoch_rewards = EpochRewards::get()?;
+    }
+
+    Ok(())
+}
+
+#[tokio::test]
+async fn get_epoch_rewards_sysvar() {
+    let program_id = Pubkey::new_unique();
+    let program_test = ProgramTest::new(
+        "epoch_reward_sysvar_getter",
+        program_id,
+        processor!(epoch_reward_sysvar_getter_process_instruction),
+    );
+
+    let mut context = program_test.start_with_context().await;
+    context.warp_to_slot(42).unwrap();
```

Comment:
Is this warp needed? if so, could we use a non-hardcoded value, like `epoch_schedule.first_normal_slot - 1` to be clear that it should be out of the reward period?

Reply:
yeah. fixed.